decoder/stats.go

```diff
@@ -365,6 +412,156 @@ func updatePokemonStats(old *Pokemon, new *Pokemon, areas []geo.AreaName) {
 	}
 }
 
+func updateRaidStats(old *Gym, new *Gym, areas []geo.AreaName) {
```

Comment:
Can you pls check your DB, unfortunately for raid stats I only get "world" entries, but not proper area entries. For invasion it works quite well :)

Reply:
I had that bug earlier under development and honestly though I had fixed it, as I havn't been able to trigger it again.
Will give it a look and see if I can trigger the behavior. Confused me a bit at the time, and somehow it works on my end right now.